src/components/ContentPreview.tsx

```diff
@@ -85,6 +90,19 @@ const ContentPreview: FC<ContentPreviewProps> = ({ resource }) => {
           scrolling="no"
         />
       )}
+      {(presentationMode === SupportedFileTypes.Youtube ||
```

Comment:
Kjørte mot prod for å sjekke at det fungerte. 
Youtube: http://localhost:3000/resource/d39d7ae7-308a-49d2-8052-9416a982bd5e
kaltura: http://localhost:3000/resource/2b803717-e280-4825-bdf2-fbd22d33878d
vimeo: http://localhost:3000/resource/9e576880-af90-47ee-b4d1-bd95e4e3711a
MediaSite: http://localhost:3000/resource/1cc3312a-30eb-470a-bf9b-f5bfd2fb0000
link: http://localhost:3000/resource/838437fe-e42e-4f80-9bad-5ba4a057bc7b

Reply:
Spotify ressurs på dev: http://localhost:3000/resource/ea405ccf-ea42-496b-8990-5ddb120e73c7